framework/grid/GridView.php

```diff
@@ -345,13 +349,15 @@ public function renderItems()
         $columnGroup = $this->renderColumnGroup();
         $tableHeader = $this->showHeader ? $this->renderTableHeader() : false;
         $tableBody = $this->renderTableBody();
-        $tableFooter = $this->showFooter ? $this->renderTableFooter() : false;
+	    $tableFooter = $this->showFooter ? (!$this->placeFooterAfterBody ? $this->renderTableFooter() : false) : false;
```

Comment:
It's better not to use short conditions in case there are many. Let's have it as proper `if`-s.